doc/manuals/orion-api.md

```diff
@@ -1047,15 +1055,30 @@ The values that `options` parameter can have for this specific request are:
 | `values`    | when used, the response payload uses the `values` simplified entity representation. See [Simplified Entity Representation](#simplified-entity-representation) section for details.                                |
 | `unique`    | when used, the response payload uses the `values` simplified entity representation. Recurring values are left out. See [Simplified Entity Representation](#simplified-entity-representation) section for details. |
 
-**Response**
+_**Request headers**_
+
+| Header               | Optional | Description                                                                                    | Example   |
+|----------------------|----------|------------------------------------------------------------------------------------------------|-----------|
+| `Fiware-Service`     | ✓        | Tenant or service. See subsection [Multitency](#multitenancy) for more information.            | `acme`    |
+| `Fiware-ServicePath` | ✓        | Service path or subservice. See subsection [Service Path](#service-path) for more information. | `project` |
```

Comment:
```suggestion
| `Fiware-ServicePath` | ✓        | Service path or subservice. See subsection [Service Path](#service-path) for more information. | `/project` |
```

Apply the same change to all cases.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4167/commits/7c052338831711718a02ee98c0b861790adb0de5